### PR TITLE
Add Swagger Definition

### PR DESCRIPTION
**Pull request by macrael**
I based this branch off of Rebecca's and my branch for the pop integration because it had the API package configuration code in it. I can pull this out if we are unsure about merging that, later. 

This includes a swagger definition for our API which I believe should be single source of truth for all API disputes.

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
+swagger: '2.0'
+info:
+  description: The API for my.move.mil
+  version: 0.0.1
+  title: my.move.mil
+  contact:
+    email: ppp@truss.works
+  license:
+    name: MIT
```
Comment (tenshiemi) [inline swagger.yaml]: May want to use the license we've been using elsewhere, the BSD 3-Clause License

File: README.md [hunk 2]
```diff
@@ -46,6 +46,10 @@ The above will start the server running and starts the webpack dev server, proxi
 
 Dependencies are managed by yarn
 
+### API
+
+The api is defined in a single file: ./swagger.yaml and served at /api/v1/swagger.yaml. it is the single source of truth for what the API contract between client and server should be.
```
Comment (tenshiemi) [inline README.md]: I kinda think it makes sense to have it in the /server folder at least, but it's just personal preference

File: swagger.yaml (hunk 1 above)
Comment (macrael) [reply swagger.yaml]: Where are we using BSD? The license at the top level is MIT. (well, actually mostly I think it's not even eligible for copyright)

Comment (tenshiemi) [reply swagger.yaml]: In all our terraform modules. Top level should probably be updated.

**Comment by macrael** [comment]
I'm deleting this PR for now. Branch is still good if we want to move forward with swagger.

**Pull request closed without merging**